### PR TITLE
clouds demo

### PR DESCRIPTION
**Pull request by samthor**
@alancutter 

Showcases changing `playbackRate` on lots of things.

### CODE REVIEW

File: clouds/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  var cloud = createCloud();
+  document.body.appendChild(cloud);
+  var player = animateCloud(cloud);
+  players.push(player);
+}
+
+var playbackRate = 1;
+var maxPlaybackRate = 10;
+
+document.body.addEventListener('wheel', function(ev) {
```
Comment (alancutter) [inline clouds/index.html]: Scrolling may be a bit too unintuitive despite the helpful message, perhaps use the Y coordinate of mouse/touch move?

Comment (samthor) [inline clouds/index.html]: Done, PTAL. Now support scrolling as well as click/touch, and shows speed in bar.

**Comment by alancutter** [comment]
Firefox seems to be in a permanent click mode (you only have to move the mouse).
If you disable dom.animations-api.element-animate.enabled in about:config the clouds don't move at all. ):
In Chrome there's a definite lag when the playbackRate changes, the clouds stop for a frame or two during the change and it doesn't look nice.
The last two are probably bugs in Chrome/the polyfill but I consider them blockers that the animations team need to resolve before we can add this demo to our showcase.

**Comment by samthor** [comment]
I've fixed the mouse events (they're hard turns out) and a Safari/polyfill issue. PTAL.

**Comment by alancutter** [comment]
This should be ready to merge soon: https://bugs.chromium.org/p/chromium/issues/detail?id=648502#c13

**Pull request merged**